### ISSUE TITLE
TypeError: unsupported operand type(s) for &: 'str' and 'int'

### ISSUE THREAD

**Issue by wanghaisheng** [kemingy/Plane#21]
ASCII = build_new_regex('ascii', r'[a-zA-Z0-9]+', ' ')
  File "D:\Program Files\anaconda3\lib\site-packages\plane\func.py", line 21, in build_new_regex
    PATTERNS[name] = compile_regex(regex)
  File "D:\Program Files\anaconda3\lib\site-packages\plane\func.py", line 27, in compile_regex
    expression = re.compile("(?P<%s>%s)" % (regex.name, regex.pattern), regex.flag) 
  File "D:\Program Files\anaconda3\lib\re.py", line 252, in compile
    return _compile(pattern, flags)
  File "D:\Program Files\anaconda3\lib\re.py", line 304, in _compile
    p = sre_compile.compile(pattern, flags)
  File "D:\Program Files\anaconda3\lib\sre_compile.py", line 764, in compile        
    p = sre_parse.parse(p, flags)
  File "D:\Program Files\anaconda3\lib\sre_parse.py", line 948, in parse
    p = _parse_sub(source, state, flags & SRE_FLAG_VERBOSE, 0)
TypeError: unsupported operand type(s) for &: 'str' and 'int'

**Comment by wanghaisheng** [comment]
python 3.8.5

**Comment by kemingy** [comment]
```python
Signature: build_new_regex(name, regex, flag=0, repl=' ')
Docstring:
:param str name: regex pattern name
:param str regex: regex
:param str repl: replacement

build regex pattern, space :code:`' '` in name will be replaced by
:code:`'_'`
```

I guess what you want is:
```python
ASCII = build_new_regex('ascii', r'[a-zA-Z0-9]+', repl=' ')
```

**Comment by wanghaisheng** [comment]
nope.I have no idea  ,just copy your example and run it
```


from plane import extract, build_new_regex, CHINESE_WORDS
ASCII = build_new_regex('ascii', r'[a-zA-Z0-9]+', ' ')
WORDS = ASCII + CHINESE_WORDS
print(WORDS)

>>> Regex(name='ascii_Chinese_words', pattern='[a-zA-Z0-9]+|[\\U00004E00-\\U00009FFF\\U00003400-\\U00004DBF\\U00020000-\\U0002A6DF\\U0002A700-\\U0002B73F\\U0002B740-\\U0002B81F\\U0002B820-\\U0002CEAF\\U0002CEB0-\\U0002EBEF]+', repl=' ')

``

**Comment by kemingy** [comment]
I see. Thanks for your feedback. Fixed in #22
